src/react_native/red_black_tree.cljs

```diff
@@ -0,0 +1,51 @@
+(ns react-native.red-black-tree
+  (:refer-clojure :exclude [remove update find])
+  (:require ["functional-red-black-tree" :as rb-tree]))
+
+(def tree ^js rb-tree)
+
+(defn find
+  [^js tree item]
+  (.find tree item))
+
+(defn insert
```

Comment:
Really nice to see this refactor, but IMO this namespace has nothing to do with React Native. Putting it inside `react_native/` implies there's some relationship, but there's none. How about we move it to `src/utils/red_black_tree.cljs`?

Reply:
really good point thanks